tests/integration/ovs_test.py

```diff
@@ -687,7 +687,6 @@ def test_ovsdb_global_config_add_delete_mapping():
             {
                 Ovn.BridgeMappings.LOCALNET: "net1",
                 Ovn.BridgeMappings.BRIDGE: "br1",
-                Ovn.BridgeMappings.STATE: "present",
```

Comment:
ditto: maybe we should keep one test w/ present behind, since IMO we are interested in not breaking existing users with existing configurations out there.

Reply:
I would like to break this API since this only exists for a short time. Better drop it now than too late.

Reply:
we have automated testing and docs already written using this API. 

If we break the API now, this just means we have to go back and amend all those.

Please re-consider.

Reply:
New patch only deprecate the use of `state: present`, still support it.